cmd/serve/main.go

```diff
@@ -17,6 +19,7 @@ func main() {
 		log.Fatal(err)
 	}
 	http.Handle("/", http.FileServerFS(content))
+	http.HandleFunc("/api/v1/user", http.HandlerFunc(handler_v1.CreateUser))
```

Comment:
this might need to be before the "/" handler, did you test it out?

Reply:
Seems to work fine but I'll move it just in case